src/core_plugins/kibana/public/visualize/editor/styles/_editor.less

```diff
@@ -128,6 +128,13 @@
     }
   }
 
+  .vis-editor-full-options {
+    display: flex;
+    flex-direction: column;
+    flex: 1 0 auto;
+    padding: 0px;
+  }
+
```

Comment:
This needs to be added to `.visualization-options`: 

https://github.com/ppisljar/kibana/blob/ebd3926f934dfaecd0b4637a8b246cff7ce9a84b/src/core_plugins/kibana/public/visualize/editor/styles/_editor.less#L25-L34

or override `.visualization-options` inside `.vis-editor-full-options` with those settings.